docs/guides/using_black_with_other_tools.md

```diff
@@ -210,41 +210,20 @@ mixed feelings about _Black_'s formatting style.
 #### Configuration
 
 ```
-disable = C0326, C0330
 max-line-length = 88
 ```
 
 #### Why those options above?
 
-When _Black_ is folding very long expressions, the closing brackets will
-[be dedented](../the_black_code_style/current_style.md#how-black-wraps-lines).
-
-```py3
-ImportantClass.important_method(
-    exc, limit, lookup_lines, capture_locals, callback
-)
-```
-
-Although this style is PEP 8 compliant, Pylint will raise
-`C0330: Wrong hanging indentation before block (add 4 spaces)` warnings. Since _Black_
-isn't configurable on this style, Pylint should be told to ignore these warnings via
-`disable = C0330`.
-
-Also, since _Black_ deals with whitespace around operators and brackets, Pylint's
-warning `C0326: Bad whitespace` should be disabled using `disable = C0326`.
-
-And as usual, Pylint should be configured to only complain about lines that surpass `88`
-characters via `max-line-length = 88`.
+Pylint should be configured to only complain about lines that surpass `88` characters
```

Comment:
Might be worth adding a small line/note that this configuration is only applicable latest versions, or to be more precise `>= 2.6.0`.

Reply:
Makes sense, can see it in the [2.6.0](https://pylint.pycqa.org/en/latest/whatsnew/changelog.html?highlight=max-line-length#what-s-new-in-pylint-2-6-0) release notes.

Given there have been a good number of releases since 2.6.0, do you think this can just be something like
```
N.B. if using `pylint < 2.6.0`, also disable `C0326` and `C0330`.
```
or would you rather keep the longer descriptions?

Reply:
I think a short note like that is good. We can skip the details.

Reply:
Added the old error code callout in now 😄